src/pal/src/include/pal/threadsusp.hpp

```diff
@@ -73,6 +73,8 @@ Module Name:
 #include <sys/sem.h>
 #include <sys/types.h>
 
+#elif __ANDROID__
+#include <semaphore.h>
```

Comment:
nit: `#elif HAVE_SEMAPHORE_H` (in contrast to `HAVE_SYS_SEMAPHORE_H` above), that way it will also take care of future platforms.